drivers/scheduler/k8s/k8s.go

```diff
@@ -2613,10 +2613,14 @@ func (k *K8s) ValidateVolumes(ctx *scheduler.Context, timeout, retryInterval tim
 			}
 		} else if obj, ok := specObj.(*v1.PersistentVolumeClaim); ok {
 			err := k8sCore.ValidatePersistentVolumeClaim(obj, timeout, retryInterval)
-			if err != nil && !options.ExpectError {
-				return &scheduler.ErrFailedToValidateStorage{
-					App:   ctx.App,
-					Cause: fmt.Sprintf("Failed to validate PVC: %v. Err: %v", obj.Name, err),
+			if err != nil {
+				if options != nil && options.ExpectError {
+					// ignore
+				} else {
```

Comment:
nit: We could also write it like this - 
```
if err != nil && (options == nil || !options.ExpectError) {
    return &scheduler.ErrFailedToValidateStorage{...}
}
```

Reply:
yep, I could do that too, it just a bit easier to read for me in this way